backend/service/chaos/experimentation/experimentstore/experiment_store.go

```diff
@@ -148,14 +148,18 @@ func (fs *experimentStore) CancelExperimentRun(ctx context.Context, id uint64) e
 
 // GetExperiments experiments with a given type of the configuration. Returns all experiments if provided configuration type
 // parameter is an emtpy string.
-func (fs *experimentStore) GetExperiments(ctx context.Context, configType string) ([]*experimentation.Experiment, error) {
-	sql := `
-        SELECT experiment_run.id, details FROM experiment_config, experiment_run
-			WHERE
-				experiment_config.id = experiment_run.experiment_config_id
-				AND ($1 = '' OR $1 = experiment_config.details ->> '@type')`
-
-	rows, err := fs.db.QueryContext(ctx, sql, configType)
+func (fs *experimentStore) GetExperiments(ctx context.Context, configType string, status experimentation.GetExperimentsStatus) ([]*experimentation.Experiment, error) {
+	query := `
+        SELECT 
+			experiment_run.id, 
+			details
+		FROM experiment_config, experiment_run
+		WHERE
+			experiment_config.id = experiment_run.experiment_config_id
+			AND ($1 = '' OR $1 = experiment_config.details ->> '@type')
+			AND ($2 != 'RUNNING' OR (experiment_run.cancellation_time is NULL AND NOW() > lower(experiment_run.execution_time) AND (upper(experiment_run.execution_time) IS NULL OR NOW() < upper(experiment_run.execution_time))))`
```

Comment:
Could you please add comments for each WHERE clause and explain what is being done here?